js/ad_manager_vast.js

```diff
@@ -636,11 +688,10 @@ OO.Ads.manager(function(_, $) {
     this._onVastError = function() {
       this.errorType = 'directAjaxFailed';
       this._ajax(this._getProxyUrl(), this._onFinalError, 'script');
-      this.trigger(this.ERROR, this);
```

Comment:
This will be part of the error handling code that Douglas will be checking in. Should I include a temporary one for now?